src/test/java/com/example/postgresdemo/controller/QuestionControllerTest.java

```diff
@@ -0,0 +1,101 @@
+package com.example.postgresdemo.controller;
+
+import com.example.postgresdemo.model.Question;
+import com.example.postgresdemo.repository.QuestionRepository;
+
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultMatcher;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.web.context.WebApplicationContext;
+
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.client.match.MockRestRequestMatchers.jsonPath;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+
+@AutoConfigureMockMvc
+public class QuestionControllerTest {
+    @Autowired
+    private QuestionRepository questionRepository;
+
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private QuestionController questionController;
+
+    @Test
+    public void testCreateMockMvc() {
+        assertNotNull(mockMvc);
+    }
+
+    private boolean fillQuestions(Integer number) {
```

Comment:
You could use `@BeforeEach` junit annotation for this method ([see how it works](https://howtodoinjava.com/junit5/before-each-annotation-example/)).